### ISSUE TITLE
[Bug Report][3.3.4] Menu open in hover not work in mobile device.

### ISSUE THREAD

**Issue by alokVishu** [vuetifyjs/vuetify#17640]
### Environment
**Vuetify Version:** 3.3.4
**Vue Version:** 3.3.4
**Browsers:** Chrome 114.0.0.0
**OS:** Windows 10

### Steps to reproduce
Check the menu with open in hove in mobile device

### Expected Behavior
It should be working on mobile.

### Actual Behavior
Dropdown is not seen in mobile device

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqtVMFunDAQ/ZWRL2mlNUhNT2izStVe+wUhBxYmirvGtuxZkhXi3zs2uxtDVCWHnmDmPd57MwYeRvHDuWI4oqjElrB3uiHc1QZg26kBWt2EcFcLwleSLRpCX4sEM2GQPZrjXABYh0ZaI5/twKTUPBOZelGGQQZtqWpaUkND1rP2CM5bF2C6Kp/V92TeaoDW6sR3XvWNP9UiB5mtTJdQ1sqxTBPgF6Odfcl0t2UyeotaZlu4NgepVaBlvNiRitnLHE8p5JcIbIAz4etXvkCsF7kAqgOemJo4/0ycG0lSpHE3jkmtSBVMUxxhzVkOeEWzMRcjxTIe5nzyJR89361WsQ2tV44gIB1dIrbWBJongzt4mKVGSAkquPmpVXuA33gD0+Z/Y/Dtgj7GoHO0ndgIrcwhFH+CNfxGj5FQizbw5qtLwFo8E7lQlWXbGWZ2qNXgC4NUGteX/C2Qejrd3xa3xXfeRKBLS+pmH4okdjae2FD1znqSfeNWrjOQnFMn9lgp1h9EuGda6Y+GVI+ys32eJWsXGHq59/YloGeRWpy3lWxi4E9YfTQtW7yTLtmPv3Hpkd9cH/8In5to9VjuuYLeTRbdp7jvaSPSY7z4+fr4F7mFgBc=)

https://github.com/vuetifyjs/vuetify/assets/80033712/1a01fcb9-90e7-4b15-bdec-7f53278b72f4

**Comment by limsocheat** [comment]
I have the same issue, do we have solution for this issue?

**Comment by hayreenfly** [comment]
any updates on this?

**Comment by tzezar** [comment]
Same problem on brave browser

**Comment by icanhearyousmile** [comment]
Has there been an update on this? I am still experiencing this issue in Chrome on Windows 10.

**Comment by RunningDeer** [comment]
same problem

**Comment by RunningDeer** [comment]
@KaelWD, I would appreciate update, thanks man!
